docs/es/docs/contributing.md

```diff
@@ -0,0 +1,447 @@
+
+### Instalar requerimientos utilizando pip
+
+Después de activar el entorno como se describe arriba:
+
+<div class="termy">
+
+```console
+$ pip install -r requirements.txt
+
+---> 100%
+```
+
+</div>
+
+Esto instalará todas las dependencias y tu FastAPI local en tu entorno local.
+
+### Utilizando tu FastAPI local
+
+Si creas un archivo Python que importa y usa FastAPI, y lo ejecutas con Python desde tu entorno local, usará tu código fuente local clonado de FastAPI.
+
+Y si actualizas ese código fuente local de FastAPI cuando vuelvas a ejecutar ese archivo Python, utilizará la versión nueva de FastAPI que acabas de editar.
+
+De esa manera, no tendrás que "instalar" tu versión local para poder probar cada cambio.
+
+!!! note "Detalles Técnicos"
+    Esto solo sucede cuando realizas la instalación utilizando el `requirements.txt` incluido en lugar de ejecutar `pip install fastapi` directamente.
+
+    Esto se debe a que dentro del archivo `requirements.txt`, la versión local de FastAPI está marcada para instalarse en modo "editable", con la opción `-e`.
+
+### Formatear el código
+
+Hay un script que puedes ejecutar y que formateará y limpiará todo tu código:
+
+<div class="termy">
+
+```console
+$ bash scripts/format.sh
+```
+
+</div>
+
+También ordenará automáticamente todas tus importaciones.
+
+Para ordenarlos correctamente, necesitas tener FastAPI instalado localmente en tu entorno, con el comando de la sección anterior usando `-e`.
+
+## Documentación
+
+Primero, asegúrese de configurar su entorno como se describe anteriormente, eso instalará todos los requisitos.
+
+### Documentación en vivo
+
+Durante el desarrollo local, hay un script que construye el sitio y verifica si hay cambios, recargando en vivo:
+
+<div class="termy">
+
+```console
+$ python ./scripts/docs.py live
+
+<span style="color: green;">[INFO]</span> Serving on http://127.0.0.1:8008
+<span style="color: green;">[INFO]</span> Start watching changes
+<span style="color: green;">[INFO]</span> Start detecting changes
+```
+
+</div>
+
+Servirá la documentación en `http://127.0.0.1:8008`.
+
+De esa manera, puedes editar los archivos de documentación/código fuente y ver los cambios en vivo.
+
+!!! tip "Consejo"
+    Alternativamente, puedes realizar los mismos pasos que los scripts realizan manualmente.
+
+    Ve al directorio de idiomas, para los documentos principales en inglés en `docs/en/`:
+
+    ```console
+    $ cd docs/en/
+    ```
+
+    Luego ejecuta `mkdocs` en ese directorio:
+
+    ```console
+    $ mkdocs serve --dev-addr 8008
+    ```
+
+#### Typer CLI (opcional)
+
+Las instrucciones aquí te muestran cómo utilizar el script en `./scripts/docs.py` directamente con el programa `python`.
+
+Pero también puedes usar <a href="https://typer.tiangolo.com/typer-cli/" class="external-link" target="_blank">Typer CLI</a> y obtendrás el autocompletado en tu terminal para ver los comandos después de instalar el completamiento.
```

Comment:
It is not a real word `completamiento`
```suggestion
Pero también puedes usar <a href="https://typer.tiangolo.com/typer-cli/" class="external-link" target="_blank">Typer CLI</a> y obtendrás el autocompletado en tu terminal para ver los comandos después de instalar el completado.
```

Reply:
Puedo cambiarlo... pero la palabra si existe en español:

https://dle.rae.es/completamiento

Reply:
ahh, disculpas entoncés. Tendría que haberlo buscado ahi tambien. Si preferís que quede como antes por mi OK